applications/application-life-cycle-management/odc-creating-an-application-using-the-developer-perspective.adoc

```diff
@@ -0,0 +1,26 @@
+ifdef::context[:parent-context-of-odc-creating-an-application-using-the-developer-perspective: {context}]
+[id="odc-creating-an-application-using-the-developer-perspective_{context}"]
+= Creating an application using the Developer perspective
+include::modules/common-attributes.adoc[]
+:context: odc-creating-an-application-using-the-developer-perspective
+
+toc::[]
+
+An application is an abstraction composed of one or more components. The *Developer* perspective in the web console provides you the following options from the *Add* view to create applications and associated services and deploy them on {product-title}:
```

Comment:
I don't think we need to define what an application is here. Perhaps remove the first sentence.